recipes/napkin/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "napkin" %}
+{% set version = "0.6.8" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 2bd3352f588b4005a9e3dc20d39d338838dd8926ff85ddd4629dfd04009a8fe6
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - napkin = napkin.cli:main
+    - napkin_plantuml = napkin.plantuml_cli:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3
+    - requests
+  run:
+    - python >=3
+    - requests
+
+test:
+  imports:
+    - napkin
+    - tests
+  commands:
+    - napkin --help
+    - napkin_plantuml --help
```

Comment:
```suggestion
    - napkin_plantuml --help
    - pip check
  requires:
    - pip
```

Reply:
Thank you @marcelotrevisani I added your suggestion